imgix/vue
# build: use vite for bundling and remove vue cli

# Description

## Before

- Application was bundled and test with vue-cli (deprecated)

## After
- Application is bundled with vite and tested with vitest

## Steps to test
- See the `next` branch running on [CodeSandbox](https://codesandbox.io/p/github/imgix/vue/next?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522gitSidebarPanel%2522%253A%2522COMMIT%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522panelType%2522%253A%2522TABS%2522%252C%2522id%2522%253A%2522clhfg72q900093b6kukvlar02%2522%257D%252C%257B%2522type%2522%253A%2522PANEL%2522%252C%2522panelType%2522%253A%2522TABS%2522%252C%2522id%2522%253A%2522clhfgfnsn00mf3b6kz8bllz4y%2522%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clhfg72q900093b6kukvlar02%2522%253A%257B%2522id%2522%253A%2522clhfg72q900093b6kukvlar02%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clhfg72q900083b6kur1ptlii%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%257D%252C%257B%2522id%2522%253A%2522clhfg7y9600mu3b6kbfxwt4d4%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clhfg82ja0006f3hy1afpe3ec%2522%257D%252C%257B%2522id%2522%253A%2522clhfgf67m006d3b6kesrz24k2%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clhfg72q900083b6kur1ptlii%2522%257D%252C%2522clhfgfnsn00mf3b6kz8bllz4y%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522clhfgf9de00c03b6kyvx48bms%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clhfgfnsn00mf3b6kz8bllz4y%2522%252C%2522activeTabId%2522%253A%2522clhfgf9de00c03b6kyvx48bms%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)
OR
- In your Vue application `yarn add @imgix/vue@3.1.0-rc.1` or `yarn add @imgix/vue@next`
- [Configure](https://github.com/imgix/vue#standard-vue-3-app) the imgix plugin
- Try using `<ix-img src="https://sdk-test.imgix.net/amsterdam.jpg" sizes="100vw" />`

## Checklist

- [x] All existing unit tests are still passing
